client/src/components/Approval.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Alert from './Alert';
+
+function Approval(approved) {
+  //   const [approval, setApproval] = React.useState(false);
+  //   React.useEffect(() => {
+  //     setApproval(true);
+  //   });
+  if (approved === true) {
+    return <Alert>Email Successfully Sent 🎁</Alert>;
```

Comment:
I would have used the Confirmation component, as the Alert has been used for warnings so far. (The alert has a red border whereas the confirmation has a green one.)

Reply:
Thank you, we changed that :)